lib/chatty_student.rb

```diff
@@ -0,0 +1,14 @@
+class ChattyStudent < Student
+  def hello
+    super
+    puts "How are you doing today? I'm okay, but I'm kind of tired. Did you watch The Walking Dead last night? You didn't?! Oh man, it was so crazy! What, you don't want any spoilers? Okay well let me just tell you who died..."
+  end
+
+  def raise_hand
+    count = 10
+    while count > 0
+      super
+      count -= 1
+    end
```

Comment:
```suggestion
    10.times { super }
```

Reply:
I figured I could use times here but wasnt sure exactly. Thanks for showing me